### ISSUE TITLE
Metrics/BlockLength (and alike) should be able to ignore multi-line method calls

### ISSUE THREAD

**Issue by toaster** [rubocop/rubocop#9533]
## Is your feature request related to a problem? Please describe.

I’m always frustrated when RuboCop complains when I changed a method call with multiple parameters to span multiple lines and thus broke the block length limit.
While it helps to use keyword arguments and add `hash` to `CountAsOne` this is not yet perfect.

For example, the statement
```ruby
foo(
  arg1: 1,
  arg2: 2,
  arg3: 3,
)
```
counts as 3 lines, but
```ruby
foo({
  arg1: 1,
  arg2: 2,
  arg3: 3,
})
```
only counts as 1.

## Describe the solution you'd like

I’d like an option for `CountAsOne` like `method` or alike which allows to break a method call onto multiple lines without counting towards the block length limit.

## Describe alternatives you've considered

For keyword args it helps to use `hash` in `CountAsOne` but the start and end line are still counted unless you explicitly wrap the args into a hash.
For positional args one could wrap them into `*[…]` but this lets `Lint/RedundantSplatExpansion` complain.

## Additional context

Probably the same should be added to `Class-`, `Method-` and `ModuleLength`.
